wp-block-debug.php

```diff
@@ -0,0 +1,57 @@
+<?php
+/**
+ * Plugin Name: Alley Block Debug
+ * Plugin URI: https://github.com/alleyinteractive/wp-block-debug
+ * Description: Block debug panel.
```

Comment:
Would love to see a little more detail on what this is all about here and in `readme.md` in a future PR!

Reply:
I see it in #2!